src/830into.js

```diff
@@ -203,7 +203,8 @@ alasql.into.CSV = function(filename, opts, data, columns, cb) {
 	data.forEach(function(d){
 		s += columns.map(function(col){
 			var s = d[col.columnid];
-			s = (s+"").replace(new RegExp('\\'+opt.quote,"g"),'""');
+			var quotereg = (opt.quote === '') ? '' : '\\' + opt.quote;
```

Comment:
Having no quote represent that no replcaement is needed - so i feel it would be better if `opt.quote` was a test condition of if replacement is needed